learning-team/2023/2023-10-02.md

```diff
@@ -0,0 +1,94 @@
+# 2nd October 2023
+
+Facilitator & Note Taker: Chris Manson
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+
+- [x] Chris Manson (CM)
+- [x] Jared Galanis (JG)
+- [x] Jen Weber (JW)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+- [ ] Chris Ng (CN)
+- [x] Anne-Greeth Schot-van Herwijnen (AG)
+
+### Guests
+
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+[Open pull requests](https://help-wanted.emberjs.com/pull-requests)
+
+[RFC review app](https://rfcs.emberjs.com/)
+
+[RFC PR Queue](https://github.com/emberjs/rfcs/pulls)
+
+## Topics
+
+### API docs search (JG) 
+
+- JG: I'm still planning to do this
+- JW: if you don't get to it in the next few days can we write an issue
+- CM: lets do that now
+- https://github.com/ember-learn/ember-api-docs-data/issues/14#issuecomment-1743235069
+- this is the "smallest possible change" that could fix the situation right now
+
+
+### AI Experiment with the help bot (JW, 10mins)
+
+- JW: NullVoxPopuli is trying out an AI help bot
+- startup is called [glime](https://glimelab.ai/)
+- they are training it on api docs and it also reads stack overflow
+- it's in a private channel while we evaluate it
+- it's not right all the time, but it gives you clues and cites its sources
+- NullVoxPopuli is testing with a few others people and then we'll open it to a wider group for feedback
+- JW: how do we appropriatly message this
+- CM: are we able to get a list of questions asked so we could see what things are lacking in our own docs?
+- JW: I'll ask the group
+- this could also point to places that are potenitally lacking in Ember or in the guides
+- CM: this implies that we would need to audit the answers so we would know if they are good or bad answers
+- JW: we might be due another wave of "stack overflow fixing" 
```

Comment:
I did notice an improvement after _removing_ stack overflow from its dataset.

I think stackoverflow _could_ be helpful, but only if we can scope it to _very recent_ sets of answers